addons/kotsadm/alpha/install.sh

```diff
@@ -287,19 +285,6 @@ EOF
     rm kotsadm.cnf kotsadm.key kotsadm.crt
 }
 
-# TODO rotate
```

Comment:
This was used for deleting nodes directly from kotsadm UI. Now we exec into ekc-operator pod to do that.